pkg/ddc/alluxio/transform_test.go

```diff
@@ -1033,3 +1033,163 @@ func TestTransformFuseProperties(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateNonNativeMountsInfo(t *testing.T) {
+	engine := &AlluxioEngine{Log: fake.NullLogger()}
+	var x int64 = 1000
+	ctrl.SetLogger(zap.New(func(o *zap.Options) {
+		o.Development = true
+	}))
+
+	type testCase struct {
+		Name      string
+		DataSet   *datav1alpha1.Dataset
+		wantValue []string
+	}
+
+	testCases := []testCase{
+		{
+			Name: "generate non native mount infos",
+			DataSet: &datav1alpha1.Dataset{
+				Spec: datav1alpha1.DatasetSpec{
+					Mounts: []datav1alpha1.Mount{{
+						MountPoint: "https://mirrors.bit.edu.cn/apache/hbase",
+						Name:       "hbase",
+						Shared:     true,
+						ReadOnly:   true,
+					}, {
+						MountPoint: "oss://oss.com/test",
+						Name:       "oss",
+						EncryptOptions: []datav1alpha1.EncryptOption{
+							{
+								Name: "passwd",
+								ValueFrom: datav1alpha1.EncryptOptionSource{
+									SecretKeyRef: datav1alpha1.SecretKeySelector{
+										Name: "ds-secret",
+										Key:  "secret-key",
+									},
+								},
+							},
+						},
+					}, {
+						MountPoint: "local:///mnt/test",
+						Name:       "local",
```

Comment:
Please define a constant instead of duplicating this literal "local:///mnt/test" 3 times.

Reply:
fixed.